README.md

```diff
@@ -15,4 +15,5 @@ ruby library or CVS via cvs specialized library.
 
 The third layer is model of file that allows more highlevel actions and its main
 purpose is to ensure consistency of file and providing high level API for
-file manipulation.
+file manipulation. The models live in their own gems as plugin built on top of
+this gem.
```

Comment:
The third layer consist on a set of models representing the files to provide more high level actions. Its main
 purposes are to ensure consistency of the files and to provide high level API for manipulating the files. The models live in their own gems as plugins built on top of this gem.